settings.py

```diff
@@ -44,6 +44,7 @@
 SERVICES_URL = SITE_URL
 
 CODE_MANAGER_URL = os.environ.get('CODE_MANAGER_URL') or 'http://localhost:3000'
+CODE_MANAGER_DOMAIN = urlparse(CODE_MANAGER_URL).netloc
```

Comment:
I'm not sure about having both `CODE_MANAGER_URL` and `CODE_MANAGER_URL` settings - can't you just do `urlparse(settings.CODE_MANAGER_URL).netloc` when you need the domain in `_is_safe_url`?

Reply:
Yep, I guess I could do that.